PWGHF/DataModel/ReducedDataModel.h

```diff
@@ -0,0 +1,249 @@
+DECLARE_SOA_COLUMN(SigTglZ, cTglZ, float);      //! Covariance matrix
+DECLARE_SOA_COLUMN(SigTglSnp, cTglSnp, float);  //! Covariance matrix
+DECLARE_SOA_COLUMN(SigTgl2, cTglTgl, float);    //! Covariance matrix
+DECLARE_SOA_COLUMN(SigQ2PtY, c1PtY, float);     //! Covariance matrix
+DECLARE_SOA_COLUMN(SigQ2PtZ, c1PtZ, float);     //! Covariance matrix
+DECLARE_SOA_COLUMN(SigQ2PtSnp, c1PtSnp, float); //! Covariance matrix
+DECLARE_SOA_COLUMN(SigQ2PtTgl, c1PtTgl, float); //! Covariance matrix
+DECLARE_SOA_COLUMN(SigQ2Pt2, c1Pt21Pt2, float); //! Covariance matrix
+
+DECLARE_SOA_COLUMN(Px, px, float); //! Momentum in x-direction in GeV/c
+DECLARE_SOA_COLUMN(Py, py, float); //! Momentum in y-direction in GeV/c
+DECLARE_SOA_COLUMN(Pz, pz, float); //! Momentum in z-direction in GeV/c
+} // namespace hf_track_par_cov
+
+// general columns
+#define HFTRACKPARCOV_COLUMNS     \
+  hf_track_par_cov::X,            \
+    hf_track_par_cov::Alpha,      \
+    hf_track_par_cov::Y,          \
+    hf_track_par_cov::Z,          \
+    hf_track_par_cov::Snp,        \
+    hf_track_par_cov::Tgl,        \
+    hf_track_par_cov::Q2Pt,       \
+    hf_track_par_cov::SigY2,      \
+    hf_track_par_cov::SigZY,      \
+    hf_track_par_cov::SigZ2,      \
+    hf_track_par_cov::SigSnpY,    \
+    hf_track_par_cov::SigSnpZ,    \
+    hf_track_par_cov::SigSnp2,    \
+    hf_track_par_cov::SigTglY,    \
+    hf_track_par_cov::SigTglZ,    \
+    hf_track_par_cov::SigTglSnp,  \
+    hf_track_par_cov::SigTgl2,    \
+    hf_track_par_cov::SigQ2PtY,   \
+    hf_track_par_cov::SigQ2PtZ,   \
+    hf_track_par_cov::SigQ2PtSnp, \
+    hf_track_par_cov::SigQ2PtTgl, \
+    hf_track_par_cov::SigQ2Pt2,   \
+    hf_track_par_cov::Px,         \
+    hf_track_par_cov::Py,         \
+    hf_track_par_cov::Pz
+
+namespace hf_reduced_track_index
+{
+DECLARE_SOA_INDEX_COLUMN(HfReducedCollision, hfReducedCollision); //! ReducedCollision index
+DECLARE_SOA_INDEX_COLUMN(Track, track);                           //! Track index
+DECLARE_SOA_COLUMN(HasTPC, hasTPC, bool);                         //! Flag to check if track has a TPC match
+DECLARE_SOA_COLUMN(HasTOF, hasTOF, bool);                         //! Flag to check if track has a TOF match
+} // namespace hf_reduced_track_index
+
+DECLARE_SOA_TABLE(HfReducedTracksWithSel, "AOD", "HFREDTRACKWSEL", //! Table with track information for reduced workflow
+                  soa::Index<>,
+                  hf_reduced_track_index::TrackId,
+                  hf_reduced_track_index::HfReducedCollisionId,
+                  HFTRACKPARCOV_COLUMNS);
+
+namespace hf_track_pid_with_sel
+{
+DECLARE_SOA_COLUMN(Pt, pt, float); //! Transverse momentum of the track in GeV/c
+} // namespace hf_track_pid_with_sel
+
+// table with all attributes needed to call getStatusTrackPIDTpcAndTof() in the selector task
+DECLARE_SOA_TABLE(HfReducedTracksPIDWithSel, "AOD", "HFREDTRACKPID", //! Table with PID track information for reduced workflow
```

Comment:
```suggestion
DECLARE_SOA_TABLE(HfReducedTracksPidWithSel, "AOD", "HFREDTRACKPID", //! Table with PID track information for reduced workflow
```

Reply:
I realized I could give simpler names to the Tracks tables: `HfTracksReduced` and `HfTracksPidReduced`. I initially added `WithSel` because I use `TracksPIDWithSel = soa::Join<aod::BigTracksPIDExtended, aod::TrackSelection>` in `dataCreatorDplusPiReduced.cxx` but I don't use any column of `aod::TrackSelection` beyond this task.